schema/GeneticVariant_Alt_ID_Database_properties.mcf

```diff
@@ -0,0 +1,1023 @@
+# Generated by script format_dbSNP_alt_ID_database_property_schema		.py
```

Comment:
not a big deal, but there's an interesting whitespace here

Reply:
removed